python/paddle/fluid/tests/.gitignore

```diff
@@ -1,4 +1,51 @@
+.gitignore
```

Comment:
`.gitignore` files should not be changed

Reply:
done